cpp/core/config/GlutenConfig.cc

```diff
@@ -34,12 +39,28 @@ parseConfMap(JNIEnv* env, const uint8_t* planData, const int32_t planDataLength)
   return sparkConfs;
 }
 
+std::optional<std::regex> getRedactionRegex(const std::unordered_map<std::string, std::string>& conf) {
+  auto it = conf.find(REGEX_REDACT_KEY);
```

Comment:
Please move this one to anonymous namespace.

Reply:
Done thanks!